build.sbt

```diff
@@ -430,9 +432,11 @@ releaseProcess := Seq[ReleaseStep](
   setReleaseVersion,
   commitReleaseVersion,
   tagRelease,
-  releaseStepCommandAndRemaining("+publishSigned"),
+  publishArtifacts,
```

Comment:
I am guessing that the above was a workaround for https://github.com/sbt/sbt-release/issues/214

Reply:
OK maybe I should revert that change then? I haven't tested it yet because I haven't tried to do a release with these code changes yet...